src/configs/sort.ts

```diff
@@ -89,6 +89,22 @@ export async function sortPackageJson(): Promise<FlatConfigItem[]> {
             ],
             pathPattern: '^exports.*$',
           },
+          {
+            order: [
+              // client hooks only
+              'pre-commit',
+              'prepare-commit-msg',
+              'commit-msg',
+              'post-commit',
+              'pre-rebase',
+              'post-rewrite',
+              'post-checkout',
+              'post-merge',
+              'pre-push',
+              'pre-auto-gc',
+            ],
+            pathPattern: '^(?:gitHooks|husky|simple-git-hooks)$',
```

Comment:
I am not sure whether to have this tool-specific configuration either... Also, how is this order decided?

Reply:
It comes from official git hooks page https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks